new-app/src/App/runGame/RandNum.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import * as Number from '../../model/number';
+
+const RandomNumber = () => {
+
+  const getRandomDigit = (): Number.NumberDigit => {
+    const rand = Math.floor((Math.random() * 10)) 
+    const dig = Number.allDigits
```

Comment:
`dig` feels like maybe it could be more clear if it's not renamed?

`const randomDigit = Number.allDigits[rand]`